raw-data/FEATURES/component-feature-table.csv

```diff
@@ -121,12 +121,12 @@ R,0052,0,-,-,-,-,+,+,+,0,+,0,0,-,-,-,0,0,+,+,-,-,-,0,0,0,0,0,0,0,+,-,-,-,-,-,-,-
 ƙ,0199,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,-,0,0,0,+,+,-,-,-,0,0,0,-,-,-,-,-,-,-,+,-
 ʠ,02A0,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,-,0,0,0,+,-,-,-,+,0,0,0,-,-,+,-,-,-,-,+,-
 ʛ,029B,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,-,0,0,0,+,-,-,-,+,0,0,0,+,-,+,-,-,-,-,+,-
-ʘ,0298,0,-,-,-,-,+,-,-,-,-,-,-,-,-,+,-,-,-,0,0,0,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
-ǀ,01C0,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,+,+,+,-,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
-ǁ,01C1,0,-,-,-,-,+,-,-,-,-,-,-,-,+,-,0,0,+,+,-,-,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
-ǃ,01C3,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,+,+,-,-,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
-ǂ,01C2,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,+,-,+,-,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
-‼,203C,0,-,-,-,-,+,-,-,-,-,-,-,-,-,-,0,0,+,-,-,-,-,0,0,0,0,0,0,0,-,-,-,-,-,-,-,-,+
+ʘ,0298,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+,-,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
+ǀ,01C0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+,+,+,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
+ǁ,01C1,0,0,0,0,0,0,0,0,0,0,0,0,0,+,0,0,0,+,+,-,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
+ǃ,01C3,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+,+,-,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
+ǂ,01C2,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+,-,+,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
+‼,203C,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+,-,-,-,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,+
```

Comment:
this is where click features are made more like modifiers. Basically only [+click], [+lateral], and the (labial or coronal) place features are non-zero